scripts/concurrent_task_utils.py

```diff
@@ -60,12 +101,23 @@ def run(self):
         try:
             self.output = self.func()
             if self.verbose:
-                log('LOG %s:' % self.name, show_time=True)
-                log(self.output)
-                log('----------------------------------------')
+                for stdout in self.output:
+                    if stdout.name:
+                        log('Starting %s check' % stdout.name, show_time=True)
```

Comment:
I think it is better to keep as it is because it gives the user a indication that the check is running as the output is going to be printed under this statement. And if we do what you suggested then it will show `Completed running <name> check` and then `Finished <check name>`. I think this is not a good idea.

Reply:
I don't agree with you on this. A check that is finished cannot print saying `Starting check`.

Reply:
We can change it to something "Printing errors from XYZ check".